eth_keys/utils/module_loading.py

```diff
@@ -1,9 +1,9 @@
 import operator
+from typing import (Any, Tuple)
 from importlib import import_module
 
 
-def import_string(dotted_path):
-    # type: (str) -> type
+def import_string(dotted_path: str) -> Any:
```

Comment:
Do we really need the return type to be `Any` here? We should avoid that at all costs as it makes mypy useless for anything that uses the returned value of this

Reply:
I'm leaving this as `Any` since it can import *anything*.  however, I've added the proper typing to the usage of this in `get_backend` and `get_backend_class` to enforce that they return the proper strict types of backend classes and instances.